picard/file.py

```diff
@@ -630,22 +630,22 @@ def is_saved(self):
         return self.similarity == 1.0 and self.state == File.NORMAL
 
     def update(self, signal=True):
-        new_metadata = self.new_metadata
-        names = set(new_metadata.keys())
+        metadata = self.metadata
+        names = set(metadata.keys())
```

Comment:
I think both lines could be combined:

```python
names = set(metadata) + set(self.orig_metadata)
```

Reply:
Updated that, but needs to use `|` operator not `+`